apps/charts/charts-chartjs/README.md

```diff
@@ -23,42 +23,15 @@ npm run dev
 ### Build the app
 
 ```
-npm run build
+npm run build:static
 ```
 
 ### Preview the production build
 
 ```
-npm run start
+npm run start:static
 ```
 
-## Included benchmarks
-
-All the benchmarks are included in iframes in the index page. But it may be more
-convenient to open these benchmarks with their specific files.
-
-### [Observable Plot](https://github.com/observablehq/plot)
-
-You can load this benchmark with the `/observable-plot.html` page, for example
-http://localhost:5173/observable-plot.html if you run it locally or
-http://localhost:7000/resources/charts/dist/observable-plot.html in the
-context of the speedometer.
-
-Observable Plot is D3-based and outputs SVG.
-
-When run in development mode, the page will automatically execute the 3
```

Comment:
Curious why this was removed - no longer accurate?

Reply:
charts and editors were basically two apps in one folder. 
I separated them out and only kept in each readme what is relevant to its particular app.

Reply:
Makes sense!